<?xml version="1.0"?>
<launch>
    <node name="transform_publisher" pkg="sensor_fusion" type="transform_publisher" output= "screen">
        <param name="target_frame" type="string" value="odom" />
        <param name="source_frame" type="string" value="base_link" />
    </node>
</launch>
